cargo-nextest/Cargo.toml

```diff
@@ -28,3 +28,4 @@ nextest-metadata = { version = "0.1.0", path = "../nextest-metadata" }
 owo-colors = { version = "3.2.0", features = ["supports-colors"] }
 shellwords = "1.1.0"
 supports-color = "1.3.0"
+serde_json = "1.0.79"
```

Comment:
Please add a newline to the end of this file. (I need to add lint checks for all this...)